controls/radchartview/howto/hide-zerro-values-pieseries.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Hide Zero Values Labels in PieChart
+page_title: Remove Zero Values Labels in PieChart
+description: This article described how to hide zero segments labels in PieChart
+slug: radchartview-troubleshooting-hide-zero-segments-labels
+tags: troubleshooting,howto,zero,piechart,label,pieseries,slice,hide,remove,segments
+published: True
+position: 0
+---
+
+# Hide Zero Segments Labels in PieChart 
+
+There's different ways to hide/remove the labels of the zero segments inside __PieChart__. This article will describe one of them. In a few words we are going to hide the labels by binding the __Opacity__ property of the TextBlock which holds it. For the purpose of this example, firstly we will create the following ViewModels which are use for both chart scenarios:
```

Comment:
According to the conversation with Desi we can omit the 'we' statements in this paragraph. Instead you can go directly to the purpose of the article and how to achieve it. In other words, we can use a more instructional style. 

For example:

This article describes how to hide the labels of zero segments (pie slices) in RadPieChart. To do this set the Opacity of the TextBlock presenting the label to 0.